testutil/daemon/daemon.go

```diff
@@ -246,11 +247,15 @@ func (d *Daemon) StartWithLogFile(out *os.File, providedArgs ...string) error {
 		return errors.Wrapf(err, "[%s] could not find docker binary in $PATH", d.id)
 	}
 
+	if d.pidFile == "" {
+		d.pidFile = fmt.Sprintf("%s/docker.pid", d.Folder)
```

Comment:
To me, `path.Join(d.Folder, "docker.pid")` looks more elegant than this `Sprintf`.

Reply:
🤦‍♂ yes, should've seen that 👍